content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+              "sending_domain": "bar.example.com",
+              "count": 1337,
+              "threshold": 5000,
+              "seed_start": null,
+              "duration": 30,
+              "subaccount_id": 123
+            }
+          ]
+        }
+        ```
+
++ Request - Get campaigns for subaccount
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+        X-MSYS-SUBACCOUNT: 123
+
++ Response 200 (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": [
+            {
+              "campaign": "second_campaign",
+              "sending_domain": "bar.example.com",
+              "count": 1337,
+              "threshold": 5000,
+              "seed_start": null,
+              "duration": 30,
+              "subaccount_id": 123
+            }
+          ]
+        }
+        ```
+
++ Request - Get campaigns for primary account
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+        X-MSYS-SUBACCOUNT: 0
+
++ Response 200 (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": [
+            {
+              "campaign": "my_campaign",
+              "sending_domain": "foo.example.com",
+              "count": 1256,
+              "threshold": 1256,
+              "seed_start": 1620078946682,
+              "duration": 30
+            }
+          ]
+        }
+        ```
+
+### Check Campaign Status [GET /inline-seeds/status?id={campaign_id}&sending_domain={sending_domain}]
+
+Returns read-only campaign metadata.
+Auth context (from API key) provides customer and subaccount ids.
+
++ Parameters
+    + id: `my-campaign` (string, required)
+    + sending_domain: `foo.example.com` (string, required)
+
++ Request
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Response 200 (application/json)
+
+    + Body
+        ```json
+        {
+          "results": {
+            "campaign": "my_campaign",
+            "sending_domain": "foo.example.com",
+            "count": 1256,
+            "threshold": 1256,
+            "seed_start": 1620078946682,
+            "duration": 30
+          }
+        }
+        ```
++ Response 200 (application/json)
+
+    + Body
+        ```json
+        {
+          "results": {
+            "campaign": "my_campaign",
+            "sending_domain": "foo.example.com",
+            "count": 1256,
+            "threshold": 1256,
+            "seed_start": 1620078946682,
+            "duration": 30,
+            "subaccount_id": 123
+          }
+        }
+        ```
++ Response 404 (application/json)
+
+    + Body
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Seed activity not found for campaign \"my-campaign\", sending_domain \"foo.example.com\""
+                }
+            ]
+        }
+        ```
+
+### Get Subaccount Options [GET /inline-seeds/options]
```

Comment:
Should we label these "Subaccount Options"? These options apply to the primary account, too (aka subaccount 0).